lib/common-node.js

```diff
@@ -23,6 +24,11 @@ exports.EVENT_NAMES = {
   started: 'start',
   stopped: 'stop'
 }
+exports.PEER_TYPES = {
+  http: 'http',
+  udp: 'udp',
+  websocket: 'ws'
+}
```

Comment:
Would rather just use strings directly for these. The strings are the same as the object keys anyway.